traitlets/config/tests/test_application.py

```diff
@@ -623,13 +623,79 @@ def test_show_config_json(capsys):
     cfg = Config()
     cfg.MyApp.i = 5
     cfg.OtherApp
-    
+
     app = MyApp(config=cfg, show_config_json=True)
     app.start()
     out, err = capsys.readouterr()
     displayed = json.loads(out)
     assert Config(displayed) == cfg
 
+def test_env_vars_priority(monkeypatch):
+    class App(Application):
+        a = Unicode('def').tag(config=True, envvar='MY_ENVVAR')
+        b = Unicode().tag(config=True, envvar='MY_ENVVAR')
+        aliases = {'a': 'App.a', 'b': 'App.b'}
```

Comment:
Make sure to instantiate an additional Configurable in `initialize` to test that priority resolves correctly.